apps/dispatch/src/assets/config.json

```diff
@@ -4,15 +4,15 @@
   "appNameParts": ["run", "Bio", "Simulations"],
   "logo": "assets/images/runbiosimulations-logo/logo.svg",
   "banner": "assets/images/runbiosimulations-logo/logo-white.svg",
-  "appUrl": "https://run.biosimulations.org/",
-  "apiUrl": "https://api.biosimulations.org/",
+  "appUrl": "https://run.biosimulations.dev/",
+  "apiUrl": "https://api.biosimulations.dev/",
   "email": "info@biosimulations.org",
```

Comment:
still have some hard-coded .dev links

for appUrl, apiUrl, platformAppUrl, platformApiUrl.

Reply:
Thank you for the reminder to change this back to the original value!